backend/src/controller/student.py

```diff
@@ -149,7 +150,7 @@ def update_student(student_id: int):
         if address_updated:
             student_update_status = student_service.update_student_by_id(student_id, {
                 "name": data.get("name"),
-                "deactivated_at": data.get("deactivated_at"),
+                "deactivated_at": None if data.get("active") else datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
                 "birth_date": data.get("birth_date"),
```

Comment:
- [ ] update student.md file 
- [ ] update test file for student API

Reply:
@khinezarthwe Addressed both issue. Please re-review again.